src/Database/PG/Query/Class.hs

```diff
@@ -607,3 +608,9 @@ instance
       toPrepVal i,
       toPrepVal j
     ]
+
+instance Hashable JSON where
+  hashWithSalt i = hashWithSalt i . show
+
+instance Hashable JSONB where
+  hashWithSalt i = hashWithSalt i . show
```

Comment:
`JSON` and `JSONB` are both newtype wrappers around Aeson's `Value`.

Aeson defines a `Hashable` instance for `Value` since 1.5.6.0 so we should just use `GeneralizedNewtypeDeriving` to make these derivations unless we need to apply a custom hash function.

Reply:
Oh, I didn't know about that! I've corrected it now. Thanks for the suggestion!